src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
```

Comment:
This is just a bit confusing - I'm not sure what's going on here. Be worth renaming / restructuring a few things to make it clearer what's happening e.g. the `loginPrompt` param - only understood what it is by checking the params, but maybe something like `useLoginPrompt` or some such?